CFC_DataCollector/modeinfer/pipeline.py

```diff
@@ -33,30 +34,44 @@ def __init__(self):
 
   def runPipeline(self):
     allConfirmedTripsQuery = ModeInferencePipeline.getSectionQueryWithGroundTruth({'$ne': ''})
+
+
+
     (self.modeList, self.confirmedSections) = self.loadTrainingDataStep(allConfirmedTripsQuery)
     logging.debug("confirmedSections.count() = %s" % (self.confirmedSections.count()))
     
+    #commented out for now, because the training set is not large enough and there is some
+    #issue with this query, it doesn't load any sections for me
```

Comment:
The issue is probably that there is no Backup_database in your local mongodb. It's fine to comment it out for now...